client/my-sites/domains/domain-management/edit/navigation/enhanced.jsx

```diff
@@ -492,8 +494,15 @@ class DomainManagementNavigationEnhanced extends Component {
 
 			if ( status === transferStatus.CANCELLED ) {
 				title = translate( 'Remove transfer from your account' );
+				if ( ! currentUserIsOwner ) {
+					title = translate( 'Remove transfer' );
+					description = translate( 'Only the domain owner can remove the transfer' );
+				}
 			} else {
 				title = translate( 'Cancel transfer' );
+				if ( ! currentUserIsOwner ) {
+					description = translate( 'Only the domain owner can cancel the transfer' );
+				}
 			}
```

Comment:
Suggestion to not duplicate the `description` string:
```
if ( status === transferStatus.CANCELLED ) {
	title = currentUserIsOwner
		? translate( 'Remove transfer from your account' )
		: translate( 'Remove transfer' );
} else {
	title = translate( 'Cancel transfer' );
}
if ( ! currentUserIsOwner ) {
	description = translate( 'Only the domain owner can remove the transfer' );
}
```

Reply:
The two texts change slightly, "remove" in the first case and "cancel" in the second one. :)

Reply:
Oh, you're right! Scratch my observation 😄